### PR TITLE
This just gets the courses/<course_id> to work and removes the front

### LINKED ISSUE #177: Re-do URL's to have a new landing page
As discussed in chat

/courses/<course_id> should display the current landing page that the root shows with that specific course. 

/ and /courses without an id should show the /courses/<course_id> for whatever the first course found for that user is (currently there is just one course per user). If they are in no courses, just display a message of some kind.

In the future we may have another page where the user can pick their course.

All pages without a course id should show no data after the authorization fix is in.

### PR DESCRIPTION
**Pull request by jonespm**
page. Fixes #177

Still needs some additional fixes.

### CODE REVIEW

**Comment by jonespm** [comment]
@pushyamig I think this will fix the issue with the courses that you were working on providing links for. I plan to also work on the "default course" as part of #186 and #132

File: dashboard/urls.py [hunk 1]
```diff
@@ -37,9 +37,11 @@
     # These URL's are for views, the accept an empty id
     url(r'^courses/(?P<course_id>[0-9]+|)/?grades', login_required(TemplateView.as_view(template_name='grades.html')), name="grades"),
     url(r'^courses/(?P<course_id>[0-9]+|)/?assignments', login_required(TemplateView.as_view(template_name='assignments.html')), name="assignments"),
-
     url(r'^courses/(?P<course_id>[0-9]+|)/?view_file_access_within_week', login_required(TemplateView.as_view(template_name='view_file_access_within_week.html')), name="view_file_access_within_week"),
 
+    # This is the courses catch-all
+    url(r'^courses/(?P<course_id>[0-9]+|)', login_required(TemplateView.as_view(template_name='courses.html')), name="courses"),
+
     # Thse URL's are data patterns
```
Comment (pushyamig) [inline dashboard/urls.py]: This all looks good.

Comment (zqian) [reply dashboard/urls.py]: Looks good!

**Comment by jonespm** [comment]
Thanks, I'll merge this but it's going to break the main landing page from working, now only /courses is going to work. I'll fix up the main landing page in another PR.

**Pull request merged**